src/Perl6/Compiler.nqp

```diff
@@ -7,6 +7,31 @@ class Perl6::Compiler is HLL::Compiler {
     has $!language_modifier; # Active language modifier; PREVIEW mostly.
     has $!language_revisions; # Hash of language revision letters. See gen/<vm>/main-version.nqp
     has $!can_language_versions; # List of valid language version
+    has $!rakudo-home;
+
+    method BUILD() {
```

Comment:
BTW, I still not well known with nqp limitations, but this overrides `HLL::Config::BUILD` without invoking it, if memory serves me right.